README.md

```diff
@@ -1 +1,66 @@
-# bare-minimum-api
+# Bare-Minimum API
+### [View The Deployed API Here](https://bare-minimum-api-53c62eb03bf8.herokuapp.com)
```

Comment:
Could we say "access" instead of "view" as the backend repo is essentially a server so it's not "visible"